MQTTClient-C/src/V5/MQTTV5Client.h

```diff
@@ -0,0 +1,138 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Microsoft Corporation. All rights reserved.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *   http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ *******************************************************************************/
+
+#if !defined(MQTTV5_CLIENT_H)
+#define MQTTV5_CLIENT_H
+
+#if defined(__cplusplus)
+ extern "C" {
+#endif
+
+#include "../MQTTClient.h"
+#include "V5/MQTTV5Packet.h"
+
+
+typedef struct MQTTV5UnsubackData
+{
+    enum ReasonCodes reasonCode;
+    MQTTProperties* properties;
+} MQTTV5UnsubackData;
+
+/**
+ * @brief Create an MQTT client object
+ * 
+ * @param client 
+ * @param network 
+ * @param command_timeout_ms 
+ * @param sendbuf 
+ * @param sendbuf_size 
+ * @param readbuf 
+ * @param readbuf_size 
```

Comment:
```suggestion
 * @param readbuf_size 
 * @param recvProperties
```

Reply:
Review params for this file - there are a few others missing as well

Reply:
Yes - documentation is not ready - sorry I forgot to add that limitation.